test/unit/security_test.py

```diff
@@ -172,8 +172,6 @@ def test_gh_cli_is_not_available(self):
                 "--json",
                 "id,title",
                 "1000",
-                "--state",
```

Comment:
why did you remove that?

Reply:
Yes, discussed it also with @kaklakariada. 
Without this setting, the default is used (only open issues will be searched).

Reasoning:
If the issue is closed it have been addressed, therefore it is either not reported anymore or "suppressed with rational".

Reply:
ok makes sense